src/components/forms/register-birth/register-birth-form.tsx

```diff
@@ -3,6 +3,7 @@
 import { useForm } from "@tanstack/react-form";
 import { useStore } from "@tanstack/react-store";
 import { useEffect, useState } from "react";
+import { convertToISO8601, normalizeMonthInput } from "@/lib/date-format";
```

Comment:
Can we use the `date-fns` library that's already in package.json to do this conversion and normalization?
eg. using date-fns it might be something like this
```
 format(endOfDay(new Date()), "yyyy-MM-dd'T'HH:mm:ss'Z'")
```